### PR TITLE
fix(aes): class updated to be reflective of its usage pattern of unique ivs.

### PR DESCRIPTION
**Pull request by rsmets**
## Summary
Updated the Aes class to remove the `iv` attribute as class variable as it should not have the property of being constant but rather unique to each encrypt call.

## Changes
- moved the `iv` attribute from the constructor to the class methods as parameters.

## Testing
- updated the Aes class' unit tests.
- in downstream data resolver project locally

## Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas _to a borderline excessive amount_
- [x] I have made any relevant corresponding changes to the documentation including the project readme.
- [x] I have run and tested the changes locally
- [x] If it is a core feature, I have added an appropriate amount of unit tests.
- [x] Any dependent changes have been merged and published in upstream modules

### CODE REVIEW

File: src/aes.ts [hunk 1]
```diff
@@ -36,9 +36,9 @@ export class Aes {
      * @param data Uint8Array
      * @returns Buffer
```
Comment (jacobleesinger) [inline src/aes.ts:37]: let's make sure to update this jsdoc comment to reflect the updated method signature

**Review by jacobleesinger — CHANGES_REQUESTED** [review]
One tiny change request: There's a jsdoc comment that wasn't updated to reflect the new method signature. Other than that lgtm.

File: src/aes.ts (hunk 1 above)
Comment (rsmets) [reply src/aes.ts:37]: Nice, good catch!

**Review by jacobleesinger — APPROVED** [review]
👍

**Pull request merged**